### DISCUSSION TITLE
LabeledSelect component is not working in ModelessDialog.

### DISCUSSION THREAD

**Discussion by Adu2120** [iTwin/iTwinUI-react#949] [Q&A]
I am trying to use the **LabeledSelect component inside ModelessDialog component**. It shows 
![image](https://user-images.githubusercontent.com/23209728/203278694-960c609c-c8cf-40ed-9423-8cee62263821.png)
This CSS applied on that **tippy-box** class. 

and the dropdown is not visible in the UI.
![image](https://user-images.githubusercontent.com/23209728/203279697-f404afb1-7ee2-487f-9b1f-3b14ce53c5d5.png)

Is there any way to remove that CSS or override the CSS only for that component?

**Comment by mayank99** [comment]
This was first brought up in https://github.com/iTwin/iTwinUI-react/issues/783 and recently resolved in https://github.com/iTwin/itwinjs-core/pull/4717.
